packages/auth-construct/API.md

```diff
@@ -26,6 +27,13 @@ export class AmplifyAuth extends Construct implements ResourceProvider<AuthResou
     readonly resources: AuthResources;
 }
 
+// @public
+export class AmplifyReferenceAuth extends Construct implements ResourceProvider<ReferenceAuthResources> {
+    constructor(scope: Construct, id: string, props: ReferenceAuthProps);
+    // (undocumented)
+    resources: ReferenceAuthResources;
+}

```

Comment:
Continuing https://github.com/aws-amplify/amplify-backend/pull/2076#discussion_r1800112374 .

@Amplifiyer @josefaidt please chime in whether we want/don't mind this on public surface or not.